tests/tekton-pipelines_test.go

```diff
@@ -6,157 +6,258 @@ import (
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
-
-	"github.com/kubevirt/tekton-tasks-operator/pkg/common"
-	tektontasks "github.com/kubevirt/tekton-tasks-operator/pkg/tekton-tasks"
 	pipeline "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
 	v1 "k8s.io/api/core/v1"
 	rbac "k8s.io/api/rbac/v1"
+	ssp "kubevirt.io/ssp-operator/api/v1beta1"
+	"kubevirt.io/ssp-operator/internal/common"
+	tekton_tasks "kubevirt.io/ssp-operator/internal/operands/tekton-tasks"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-var _ = Describe("Tekton-pipelines", func() {
-	Context("resource creation", func() {
+var _ = Describe("Tekton Pipelines Operand", func() {
+	var (
+		tenSecondTimeout = 10 * time.Second
+	)
+
+	BeforeEach(func() {
+		strategy.SkipSspUpdateTestsIfNeeded()
+
+		updateSsp(func(foundSsp *ssp.SSP) {
+			foundSsp.Spec.FeatureGates.DeployTektonTaskResources = false
```

Comment:
`FeatureGates` may be `nil`

Reply:
Okay, in that case we then simply just do:
```
if foundSsp.Spec.FeatureGates != nil {
   foundSsp.Spec.FeatureGates.DeployTektonTaskResources = false
}
```
Right?

Reply:
yes.

Reply:
But also check for `nil` at the place that sets it to `true`. I forgot to comment there.

Reply:
Yes, I'll add nil check everywhere in the code where we use `foundSsp.Spec.FeatureGates.DeployTektonTaskResources`.